exampleSite/content/index/embed-video.md

```diff
@@ -1,17 +0,0 @@
-+++
```

Comment:
Why remove the file?

Reply:
This is most likely a mistake from me I wasn't supposed to add it. I changed the name of the file to embed_video for consistency.